app/controllers/users_controller.rb

```diff
@@ -1,5 +1,8 @@
 class UsersController < ApplicationController
-  def show; end
+  def show
+    github_results = GithubResults.new
+    @repos = github_results.repos_limit(5)
+  end
```

Comment:
repos is an array of Repo objects, created from the github api

Reply:
excellent! You just created Ruby objects from an API. Huge accomplishment!!!